data/tribes/initialization/empire/units.lua

```diff
@@ -1337,13 +1459,26 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_donkeyfarm",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Donkey Farm
+            lore = pgettext("empire_building", "‘Asinus stramen mavult quam aurum.’<br>"..
+               "(A donkey prefers straw to gold.)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Donkey Farm
+            lore_author = pgettext("empire_building", "A donkey breeder during a shortage of wheat"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Donkey Farm
             purpose = pgettext("empire_building", "Breeds cute and helpful donkeys for adding them to the transportation system.")
          }
       },
       {
          name = "empire_farm",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Farm
+            lore = pgettext("empire_building",
+               "‘Lutius iam a publicis negotiis se removerat, ruri vivebat et senatus legati eum invenerunt"..
```

Comment:
in this case Lutius defintily made a career and should be named with rank General

Reply:
This is the story of Cincinnatus: https://www.latin.it/versione/28907 There's no title in the original either.

Reply:
Well as Lutioius is a fmaous character in the widelands empire we should stick with this Lutius here.

Reply:
I meant it to illustrate that the Romans didn't put rank before names all the time. Cincinnatus was consul and later dictator (not in the bad sense) as well. I think he was called back (from the plow :) to become dictator again.

Reply:
ok, I see the point, but maybe you need to be very famous for this so everybody knows your name and rank by heart.    
I would prefer having the rank, but now I am ok with leaving it out.